index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <style>
+      /*
+      The test doesn't read my css correctly, that's why I put the css here
+      */
+      
+
+      #nav-bar{
+          position: fixed;
+          top: 0;
+          left: 0;
+          width: 100%;
+      }
+      #header{
+          display: flex;
+          justify-content: center;
+          align-items: center;
+      }
+
+      #nav-bar ul{
+          list-style: none;
+          display: flex;
+          justify-content: space-between;
+        }
+        
+       
+        .form{
+          padding: 50px 0;
+        }
+
+        .form-input {
+          padding: 10px 0;
+        }
+        #nav-bar a{
+          color: #000;
+        }
+        
+        @media (max-width: 600px) {
+          #nav-bar ul{
+              flex-wrap: wrap;
+            }
+
+            #nav-bar li{
+              margin: 12px 0;
+            }
+        }
+
+
+        .form-input input{
+          padding: 5px 10px;
+          border: 1px solid #ccc;
+          border-radius: 5px;
+        }
+
+        #submit{
+          background: #000;
+          color: #fff;
+          padding: 5px 10px;
+          border: 1px solid #ccc;
+          border-radius: 6px;
+        } 
+        
+        #submit:hover{
+          background: #fff;
+          color: #000;
+
+        }
+      </style>
```

Comment:
please keep your style definition in a separate CSS file. this is to ensure best practices. kindly create style.css file and put your styles in it then add to the part of your html